### PR TITLE
Update non-custodial-data-sources.md

### PR DESCRIPTION
**Pull request by shelri**
https://o365exchange.visualstudio.com/Enterprise%20Cloud/_workitems/edit/2171533/

These scenarios are not supported. I did not find another piece of documentation that made sense to update here. Is this a scenario where we may need to create a new scenarios page?

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/non-custodial-data-sources.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Follow these steps to add and manage non-custodial data sources in an Advanced e
 
    > [!NOTE]
    > You can use the **SharePoint** and **Exchange** sections to add sites and mailboxes associated with a Team or Yammer group as non-custodial data sources. You have to separately add the mailbox and site associated with a Team or Yammer group.
+   >Adding a Root Site URL such as data source  "https://contoso-my.sharepoint.com/personal" or  "https://contoso-my.sharepoint.com/" will yield no results. Please use Sharepoint_All instead.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/non-custodial-data-sources.md]:
```suggestion
   > Adding a Root Site URL such as data source  "https://contoso-my.sharepoint.com/personal" or  "https://contoso-my.sharepoint.com/" will yield no results. Use Sharepoint_All instead.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shelri, kindly commit the minor suggested change. For info, in accordance with the MS Style Guide, the use of the word "please" should be avoided in docs. Many thanks.

**Comment by yogkumgit** [comment]
@shelri Please review and implement the suggested changes. Thanks

**Comment by markjjo** [comment]
@shelri PR merged

**Pull request merged**